embedded/embedded_files.hxx.in

```diff
@@ -200,13 +203,17 @@ namespace embedded_files {
       }
     };
 
-    auto f = fs.at(t_filename);
-    const auto fullpath = t_location + '/' + t_filename;
+    std::string filenameonly = t_filename;
+    size_t i = t_filename.rfind('/', t_filename.length());
+    if (i != std::string::npos) {
+      filenameonly = t_filename.substr(i+1, t_filename.length() - i);
+    }
+    const auto fullpath = t_location + '/' + filenameonly;
```

Comment:
embedded_files::extractFile was broken, so I fix it (I have a use case for it... see later)